util/info.c

```diff
@@ -411,12 +429,8 @@ int main(int argc, char **argv)
 		}
 	}
 
-	if (env) {
-		ret = print_vars();
-		goto out;
-	}
```

Comment:
Why was this moved up before the command line options are processed?

Reply:
@shefty since run() processes functionality requested by user, it seemed to me that it's better to move print_vars() into run() function

Reply:
My bad -- I missed in the changes that the functionality moved out of main() and into run().